### DISCUSSION TITLE
is it Possibe to set marker on same time with different values?

### DISCUSSION THREAD

**Discussion by zeeshan-shabbir** [tradingview/lightweight-charts#1036] [Q&A]
i read setmarker documentationt but didnt find this customization. i want to know how i can acheive this because this is my need to show marker on same time with different values . thank in advance for help.

**Comment by timocov** [comment]
I think that the best way to check it is to write a code and see whether it works or not. But as you can see, [we do even have tests for that](https://github.com/tradingview/lightweight-charts/blob/f5fa7850dfbdd242a6e71f4c945c0a506bbc5729/tests/e2e/graphics/test-cases/series-markers/series-arrow-markers.js#L34-L35), so the answer is yes, you can provide several markers with the same time.
